src/network/core/http_curl.cpp

```diff
@@ -174,7 +174,7 @@ void HttpThread()
 		 * up to a second before this callback is called. There is little we can
 		 * do about this. */
 		curl_easy_setopt(curl, CURLOPT_NOPROGRESS, 0L);
-		curl_easy_setopt(curl, CURLOPT_XFERINFOFUNCTION, +[](void *userdata, curl_off_t dltotal, curl_off_t dlnow, curl_off_t ultotal, curl_off_t ulnow) -> int {
+		curl_easy_setopt(curl, CURLOPT_XFERINFOFUNCTION, +[](void *userdata, curl_off_t /*dltotal*/, curl_off_t /*dlnow*/, curl_off_t /*ultotal*/, curl_off_t /*ulnow*/) -> int {
```

Comment:
Is commenting out the parameter names intentional here, over removal?

Reply:
I did not want to force people look up the curl documentation, just to figure out what all that stuff is. But maybe you would have to anyway :)

Reply:
I kinda like what you did here honestly